click_odoo_contrib/initdb.py

```diff
@@ -65,9 +66,15 @@ def _db_storage(self):
             IrAttachment._storage = orig
 
 
-def odoo_createdb(dbname, demo, module_names, force_db_storage):
+def odoo_createdb(dbcache, dbname, demo, module_names, force_db_storage):
+    core_module_names = set(module_names).intersection(core_addons[odoo.release.series])
```

Comment:
For getting all core module needed we should be smarter and search from dependency of module_names

Reply:
@sebastienbeau You can look at https://pypi.org/project/manifestoo/

@sbidoul I don't know if this is scheduled to integrate it here ?

Reply:
#114 🙂